addons/sourcemod/scripting/VScript_ze_kitchen.sp

```diff
@@ -252,12 +252,33 @@ public void OnChangeEggsCount(const char[] output, int caller, int activator, fl
 		g_Fly.IncrementEggCount(-1);
 }
 
-public void OnEntitySpawned(int entity, const char[] classname)
+public void OnEntityCreated(int entity, const char[] classname)
```

Comment:
do the same as here:
- https://github.com/srcdslab/sm-plugin-BossHP/blob/c14b628527d0070721554dee53a3fee74bf32b35/addons/sourcemod/scripting/BossHP.sp#L101